src/components/common/ClientIntakeForm/IntakeForms/Referrals.js

```diff
@@ -50,15 +50,15 @@ const Documents = props => {
     ),
   };
 
-  const onFinish = values => {
-    if (typeof referralIndex === 'number') {
-      props.editReferral(values, referralIndex);
-      setReferralIndex(null);
-    } else {
-      props.saveReferral(values);
-    }
-    form.resetFields();
-  };
+  // const onFinish = values => {
```

Comment:
Was this commented out because it will be used later?

Reply:
It was legacy code from a previous colleague, in case it could be used later or if it would potentially be needed I just commented it out.